app/assets/images/TravelGo_files/polygon.self-08644c000d23e711283ea74461680417f8e6c30fe6c32c3dcb7cc9395e9e3f17.js

```diff
@@ -0,0 +1,40 @@
+(function() {
+  var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
+    hasProp = {}.hasOwnProperty;
+
+  this.Gmaps.Google.Builders.Polygon = (function(superClass) {
+    extend(Polygon, superClass);
+
+    function Polygon(args, provider_options) {
+      this.args = args;
+      this.provider_options = provider_options != null ? provider_options : {};
+      this.before_init();
+      this.serviceObject = this.create_polygon();
+      this.after_init();
+    }
+
+    Polygon.prototype.create_polygon = function() {
+      return new (this.primitives().polygon)(this.polygon_options());
+    };
+
+    Polygon.prototype.polygon_options = function() {
+      var base_options;
+      base_options = {
+        path: this._build_path()
+      };
+      return _.defaults(base_options, this.provider_options);
```

Comment:
Identifier 'base_options' is not in camel case.<br>Identifier 'provider_options' is not in camel case.<br>'_' is not defined.